mount_test.go

```diff
@@ -41,3 +51,227 @@ func TestIsSystemMount(t *testing.T) {
 		}
 	}
 }
+
+func TestMajorMinorNumber(t *testing.T) {
+	devices := []string{"/dev/zero", "/dev/net/tun"}
+
+	for _, device := range devices {
+		cmdStr := fmt.Sprintf("ls -l %s | awk '{print $5$6}'", device)
+		cmd := exec.Command("sh", "-c", cmdStr)
+		output, err := cmd.Output()
+
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		data := bytes.Split(output, []byte(","))
+		if len(data) < 2 {
+			t.Fatal()
+		}
+
+		majorStr := strings.TrimSpace(string(data[0]))
+		minorStr := strings.TrimSpace(string(data[1]))
+
+		majorNo, err := strconv.Atoi(majorStr)
```

Comment:
mount_test.go:75:12: ineffectual assignment to err

You should do something with err, or just use `_`